packages/unchained-client/src/parser/thorchain.ts

```diff
@@ -240,9 +240,9 @@ export class Parser {
     })()
 
     const liquidity = (() => {
-      if (txType === 'withdraw') return { type: getLiquidityType(action.pools[0]) }
```

Comment:
refund deposit/withdraws do not have a populated `pools[0]` property in their midgard action so line below would fail - adds refund type check and adds withdraw txType check